src/util.ts

```diff
@@ -304,6 +304,14 @@ export async function screenshot(client: Client): Promise<string> {
   return screenshot.data
 }
 
+export async function pdf(client: Client): Promise<string> {
+  const {Page} = client
+
+  const pdf = await Page.printToPDF()
```

Comment:
Good catch. Passing these options did not work when I was using Chrome 59, but they work now that I've upgraded to Chrome 60. I'm on macOS Sierra.

Now that I know that this works, I'll implement the ability for the user to pass in these options.